src/main/scala/org/zalando/spearheads/innkeeper/dao/RoutesPostgresRepo.scala

```diff
@@ -37,6 +37,17 @@ class RoutesPostgresRepo @Inject() (
     }.map(_.headOption)
   }
 
+  override def getOwnerTeamForRoute(routeId: Long): Future[Option[String]] = {
```

Comment:
This method is too specific, I don't think it belongs in the Repo, but in a service. The service should call `pathRepo.selectPathForRouteId`.

Also the convention in the Repos is to name the methods with `select`